components/molecule/dropdownOption/src/index.js

```diff
@@ -73,6 +75,20 @@ const MoleculeDropdownOption = ({
       )}">`,
       endTag: '</mark>'
     })
+
+    if (highlightValue) {
+      return (
+        <div className={CLASS_ITEM}>
+          <span
+            onFocus={handleInnerFocus}
+            dangerouslySetInnerHTML={{__html: mark}}
+            className={innerClassName}
+          />
+          {children}
+        </div>
+      )
+    }
```

Comment:
would this do the job for you? so that you can remove the return on L92...

I try to avoid early returns where possible...

```suggestion
      return (
        <div className={CLASS_ITEM}>
          <span
            onFocus={handleInnerFocus}
            dangerouslySetInnerHTML={{__html: mark}}
            className={innerClassName}
          />
          {highlightValue ? children : null}
        </div>
      )
```

Reply:
I have adapted the return so that it complies with the same contract and not have a wrapper so that it is currently compatible with the current use.

Agree with what you comment and in our iteration we stopped using that class to be able to do it this form, thanks